examples/remote_procedure/mnist/run_task.py

```diff
@@ -30,12 +34,29 @@
 
 from omegaconf import DictConfig
 from typing import List, Any, Dict
+from rich import print
+
+
+def my_screening_unit_generator():
+    while True:
+        yield {"isScreeningUnit": True}
+
+
+def validate_screening_unit(unit: Unit):
+    agent = unit.get_assigned_agent()
+    if agent is not None:
+        data = agent.state.get_data()
+        annotation = data["final_submission"]["annotations"][0]
+        if annotation["isCorrect"] and annotation["currentAnnotation"] == 3:
+            return True
+    return False
 
 
 @task_script(default_config_file="launch_with_local")
 def main(operator: Operator, cfg: DictConfig) -> None:
-    tasks: List[Dict[str, Any]] = [{}] * cfg.num_tasks
+    tasks: List[Dict[str, Any]] = [{"isScreeningUnit": False}] * cfg.num_tasks
```

Comment:
this could also be reverted back to what it was and the frontend would still work since it's doing a boolean check, right?

Reply:
It's being used like this: `const NUM_ANNOTATIONS = taskData.isScreeningUnit ? 1 : 3; `